src/librados/librados_c.cc

```diff
@@ -44,15 +44,25 @@
 #endif
 
 #ifdef HAVE_ASM_SYMVER
+#ifdef HAVE_ATTR_SYMVER
```

Comment:
The reason I did it this way was to avoid defining `LIBRADOS_C_API_BASE_F` and `LIBRADOS_C_API_DEFAULT_F` twice if they do not differ.

Anyway, I have updated the definitions to not rely on the assumption that one implies the other.